portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,19 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random photo.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomizeImage() {
+  // The images directory contains 17 images, so generate a random index between
```

Comment:
Revise the comments and make it more generic. Imaging that you add a new picture to the folder in the future. Now the folder contains 18 images. You need to modify the comments as well.